### ISSUE TITLE
Radial Composite Converter Sub parameter naming issue

### ISSUE THREAD

**Issue by Khemardi** [synfig/synfig#2260]
**Describe the bug**
When the 'color' property is converted to a 'Radial Composite', there is two 'Saturation' options. The last one is supposed to be 'Alpha'

**To Reproduce**
1. Add an object to the scene, example a circle
2. In it's properties right click the 'Color' and convert it to a 'Radial Composite'
3. Open up the sub-properties to see naming issue.

**Expected behavior**
Sub parameter types should read as follows:

Luma
Saturation
Hue
Alpha

**Screenshots**
https://i.ibb.co/Kq24h4r/Radial-Composite.png

**System information:**
 - OS: [Windows]
 - OS version: Windows 10
 - Synfig version: 1.4.2

**Additional comments**
None

**Comment by Keyikedalube** [comment]
Couldn't reproduce on master branch. Must have been fixed by someone else...
![Screenshot_2021-08-10_19-50-52](https://user-images.githubusercontent.com/24214509/128884035-3d258780-aedd-443c-9e0c-44c307613c13.png)

**Comment by Khemardi** [comment]
> Couldn't reproduce on master branch. Must have been fixed by someone else...
> ![Screenshot_2021-08-10_19-50-52](https://user-images.githubusercontent.com/24214509/128884035-3d258780-aedd-443c-9e0c-44c307613c13.png)

Yea it seems some did. I just redownloaded and reinstalled and it's ok now

**Comment by Keyikedalube** [comment]
@Khemardi You reported against v1.4.2. I only checked if this issue was reproducible for Synfig built from source ie., latest development changes made in this repository. Issue wasn't reproducible. This gives more reliable info to contributors and maintainers.

I just tested it on Linux AppImage and yikes your reported issue exists in the latest stable release.

Thanks for reporting :)
We need to fix this for the next stable release #2247
